lib/defaults/defaults.go

```diff
@@ -858,3 +858,9 @@ const (
 	// when using the agentless installer
 	SSHDConfigPath = "/etc/ssh/sshd_config"
 )
+
+const (
+	// AzureInviteTokenName is the name of the default token to use
+	// when templating the script to be executed.
+	AzureInviteTokenName = "azure-discovery-token"
+)
```

Comment:
Suggestion: drop the block?

```suggestion
// AzureInviteTokenName is the name of the default token to use when templating
// the script to be executed.
const AzureInviteTokenName = "azure-discovery-token"
```